src/jit/importer.cpp

```diff
@@ -11226,6 +11226,23 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 #endif // _TARGET_64BIT_
                 }
 
+#ifdef FEATURE_CORECLR
+                if ((oper == GT_ADD) || (oper == GT_SUB) || (oper == GT_MUL) || (oper == GT_DIV) || (oper == GT_MOD))
```

Comment:
It doesn't make a lot of sense (i.e. less likely that it something that we would encounter), but for completeness, we should add `GT_UMOD` and `GT_UDIV`, as well as the logical operators. We can have  `GT_UMOD` and `GT_UDIV` in the importer, as they correspond to the IL operators `rem.un` and `div.un`.